doc/misc/specs.7.rst

```diff
@@ -125,6 +125,15 @@ Many commands take a ``<package-spec>`` parameter that selects a package for
 the operation. The ``<package-spec>`` argument is matched against package
 NEVRAs, provides and file provides.
 
+When passing a ``<package-spec>``, the user can provide additional restriction
+rules for matching the arguments. Basic version comparisons can be used for this
+purpose (=, >, <, >=, <=), like this ``<package-spec> >= <version>``.
```

Comment:
Now it looks like "NEVRA >= VERSION" would be acceptable. What about rephrasing it like "When \<package-spec\> is a package name or a provide, the user can…"?

Maybe we should also specify that \<version\> here is actually "[EPOCH:]VERSION[-RELEASE]" as specified in NEVRA Matching section.